examples/components-macos/app/dune

```diff
@@ -0,0 +1,43 @@
+(rule
+ (targets env.xcconfig)
+ (action (with-stdout-to %{targets} (progn
+  (echo "STDLIB_PATH=%{ocaml_where}\n")
+  (bash "echo BUILD_PATH=$(realpath %{workspace_root})\n")
```

Comment:
I think this doesn't work for directories. `realpath` can only be used for files:
https://www.google.com/search?client=safari&rls=en&q=get+absolute+path+of+directory&ie=UTF-8&oe=UTF-8